### ISSUE TITLE
,WCAG 1.3.1,WCAG 4.1.2,Best Practice: ARIA role should be appropriate for the element (.collection-grid-filter-group__option[role="listitem"])

### ISSUE THREAD

**Issue by zcjess** [zcjess/MO#5]
#### Issue
`ARIA role should be appropriate for the element` ([aria-allowed-role](https://dequeuniversity.com/rules/axe/4.7/aria-allowed-role?application=msftAI))
#### Target application
[For sharing | Michigan Online](https://online-staging.ai.umich.edu/collections/for-sharing/explore/)
#### Element path
fieldset:nth-child(4) > .collection-grid-filter-expand[role="listbox"] > .collection-grid-filter-group__option[role="listitem"]
#### Snippet
`<label role="listitem" class="collection-grid-filter-group__option"><input type="checkbox" value="Podcast"><span class="custom-checkbox"></span><span class="custom-checkbox-label">Podcast</span><span aria-label="(2 items)" class="collection-grid-filter-...`
Element has children which are not allowed: [role=listitem] 
ARIA role listitem is not allowed for given element
Required ARIA parent role not present: list
ARIA input fields must have an accessible name (use aria-label)

fieldset:nth-child(4) > .collection-grid-filter-expand[role="listbox"] > .collection-grid-filter-group__option[role="listitem"]

fieldset:nth-child(6) > .collection-grid-filter-expand[role="listbox"] > .collection-grid-filter-group__option[role="listitem"]
<img width="404" alt="image" src="https://github.com/zcjess/MO/assets/90011962/5645d02b-4acd-4603-b34f-f721af195875">
https://online-staging.ai.umich.edu/collections/for-sharing/explore/
<br class="Apple-interchange-newline">
#### Environment
Chrome version 115.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.39.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
